File: src/f/auxlib/rect.f
C
	SUBROUTINE RECT(X0,Y0,X1,Y1)
C
C	ROUTINE TO PLOT A RECTANGLE
C	CREATED BY D. LONG JULY, 1983	AT JPL
C
C	X0,Y0	(R) LOWER LEFT HAND CORNER
C	X1,Y1	(R) UPPER RIGHT HAND CORNER
C
C	NOTE: PEN ENDS UP DOWN AT LOWER-LEFT HAND CORNER
C
	CALL PLOT(X0,Y0,3)
	CALL PLOT(X1,Y0,2)
	CALL PLOT(X1,Y1,2)
	CALL PLOT(X0,Y1,2)
	CALL PLOT(X0,Y0,2)
	RETURN
	END

File: src/c/auxlib/rect.c

/* *** LAST REVISED ON 24-DEC-1993 07:19:36.17 */
/* *** SOURCE FILE: [LONGLIB93.SOURCES.C.AUXLIB]RECT.C */

#include "my.h"

/* Table of constant values */
static integer c__2 = 2;
static integer c__3 = 3;

/* Subroutine */ int rect_(x0, y0, x1, y1)
real *x0, *y0, *x1, *y1;
{

    /* External functions */
    extern /* Subroutine */ int plot_();

/* 	ROUTINE TO PLOT A RECTANGLE */
/* 	X0,Y0	(R) LOWER LEFT HAND CORNER */
/* 	X1,Y1	(R) UPPER RIGHT HAND CORNER */

/* 	NOTE: PEN ENDS UP DOWN AT LOWER-LEFT HAND CORNER */

    plot_(x0, y0, &c__3);
    plot_(x1, y0, &c__2);
    plot_(x1, y1, &c__2);
    plot_(x0, y1, &c__2);
    plot_(x0, y0, &c__2);
    return 0;
} /* rect_ */

